core/src/main/java/com/klaytn/caver/abi/ABI.java

```diff
@@ -89,6 +102,16 @@ public static String buildFunctionString(ContractMethod method) {
         return result.toString();
     }
 
+    public static String buildFunctionString(String methodName, List<Type> parameters) {
+        StringBuilder result = new StringBuilder();
+        result.append(methodName);
+        result.append("(");
+        String params = parameters.stream().map(Type::getTypeAsString).collect(Collectors.joining(","));
+        result.append(params);
+        result.append(")");
+        return result.toString();
+    }
+
```

Comment:
This comment has nothing to do with the content of the PR.

However, I think that if a function like `encodeContractDeploy` is implemented in caver-java, it can be used conveniently by the user.
Maybe that function can be implemented like `public static String encodeContractDeploy(ContractMethod constructor, List<Object> constructorParams)`?

I will attach a link to reference(caver-js docs) below. 
If this function is implemented in caver-java, I will add this function to the common architecture.
Please let me know your opinion about implementing this :) 

https://docs.klaytn.com/bapp/sdk/caver-js/api-references/caver.abi#encodecontractdeploy